### ISSUE TITLE
Maybe Move Screen Reader training til later

### ISSUE THREAD

**Issue by andrewnordlund** [andrewnordlund/bati-itao.github.io#11]
> I found this helpful but I think that this should have come later in the path after I learned about how screen readers are used to test and how users use them.  Maybe as part of Block C – Web Accessibility after 2 – Intro to Accessibility
